geopandas/tools/sjoin.py

```diff
@@ -75,6 +75,13 @@ def sjoin(
             " joined".format(index_left, index_right)
         )
 
+    # Check if rtree is installed...
+    if not HAS_RTREE:
+        raise ImportError(
+            f"Rtree must be installed to use sjoin\n\n"
```

Comment:
We still support python 3.5, so no f-strings.

Reply:
Done (wasn't needed anyway), cheers @martinfleis